### PR TITLE
Update config output JSON format

### PR DESCRIPTION
**Pull request by donatello**
This change allows for the JSON format of config output from the server
to show both the environment variables' and config system variables'
value.

### CODE REVIEW

**Comment by donatello** [comment]
Needs https://github.com/minio/madmin-go/pull/112

File: go.sum [hunk 1]
```diff
@@ -343,8 +343,9 @@ github.com/minio/colorjson v1.0.2/go.mod h1:JWxcL2n8T8JVf+NY6awl6kn5nK49aAzHOeQE
 github.com/minio/filepath v1.0.0 h1:fvkJu1+6X+ECRA6G3+JJETj4QeAYO9sV43I79H8ubDY=
 github.com/minio/filepath v1.0.0/go.mod h1:/nRZA2ldl5z6jT9/KQuvZcQlxZIMQoFFQPvEXx9T/Bw=
 github.com/minio/madmin-go v1.3.5/go.mod h1:vGKGboQgGIWx4DuDUaXixjlIEZOCIp6ivJkQoiVaACc=
-github.com/minio/madmin-go v1.4.15 h1:1uqNMLa2ZhU05FIxPW8bUmZ5Orbs+6Tmj5B2LFJsdRI=
-github.com/minio/madmin-go v1.4.15/go.mod h1:ez87VmMtsxP7DRxjKJKD4RDNW+nhO2QF9KSzwxBDQ98=
+github.com/minio/madmin-go v1.3.11/go.mod h1:ez87VmMtsxP7DRxjKJKD4RDNW+nhO2QF9KSzwxBDQ98=
```
Comment (donatello) [inline go.sum:346]: Not sure, why it it still needs to depend on older madmin versions

Comment (vadmeste) [reply go.sum:346]: > Not sure, why it it still needs to depend on older madmin versions
I guess you need to do 'go mod tidy --compact=1.17`

File: go.sum [hunk 2]
```diff
@@ -343,8 +341,8 @@ github.com/minio/colorjson v1.0.2/go.mod h1:JWxcL2n8T8JVf+NY6awl6kn5nK49aAzHOeQE
 github.com/minio/filepath v1.0.0 h1:fvkJu1+6X+ECRA6G3+JJETj4QeAYO9sV43I79H8ubDY=
 github.com/minio/filepath v1.0.0/go.mod h1:/nRZA2ldl5z6jT9/KQuvZcQlxZIMQoFFQPvEXx9T/Bw=
 github.com/minio/madmin-go v1.3.5/go.mod h1:vGKGboQgGIWx4DuDUaXixjlIEZOCIp6ivJkQoiVaACc=
```
Comment (donatello) [inline go.sum:343]: Thanks @vadmeste - I think it removed some older versions, but there still seems to be one more.

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**